Formula/swtpm.rb

```diff
@@ -0,0 +1,41 @@
+class Swtpm < Formula
+  desc "Software TPM Emulator based on libtpms"
+  homepage "https://github.com/stefanberger/swtpm"
+  url "https://github.com/stefanberger/swtpm/archive/refs/tags/v0.7.3.tar.gz"
+  sha256 "e856d1f5842fb3335164f02f2c545dd329efbc3416db20b7a327e991a4cd49c8"
+  license "BSD-3-Clause"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gawk" => :build
+  depends_on "libtool" => :build
+  depends_on "socat" => :build
+  depends_on "gnutls"
+  depends_on "json-glib"
+  depends_on "libtasn1"
+  depends_on "libtpms"
+  depends_on "openssl@1.1"
+
+  on_linux do
+    depends_on "expect"
+    depends_on "libseccomp"
+    depends_on "net-tools"
+  end
+
+  def install
+    system "./autogen.sh", *std_configure_args, "--with-openssl"
```

Comment:
Is there no existing configure?

Reply:
I think we also need to add tss-user and tss-group for OSX. This is the official configuration https://github.com/stefanberger/swtpm/wiki#compile-on-os-x

Reply:
No, I mean a `./configure` file

Reply:
> No, I mean a `./configure` file

Yes, there is a `./configure` file. Do you mean to use `./configure` directly?

Reply:
Yes, since `autogen.sh` will generally only generate a `./configure` file anyway

Reply:
I have updated